lib/src/widgets/message_container.dart

```diff
@@ -40,6 +40,15 @@ class MessageContainer extends StatelessWidget {
   /// A flag which is used for assiging styles
   final bool isUser;
 
+  /// Provides a list of buttons to allow the usage of adding buttons to
+  /// the bottom of the message
+  final List<Widget> buttons;
+
+  /// [messageButtonsBuilder] function takes a function with this
+  /// structure [List<Widget> Function()] to render the buttons inside
+  /// a row.
+  final List<Widget> Function() messageButtonsBuilder;
```

Comment:
Also having `messageButtonsBuilder` Function doesn't make sense it could be just a `List<Widget>`

Reply:
i was thinking maybe the user wanted to do some calculation depending on the message `customProperties`